frontend/src/lib/api-server.ts

```diff
@@ -0,0 +1,109 @@
+import { Project, Thread } from '@/lib/api';
```

Comment:
Explanation: These two functions `getThread` and `getProject` are added to be used in the file above `app/share/[threadId]/layout.tsx` to get project's meta details

These are server side functions and different from same same functions in `src/lib/api.ts`